### ISSUE TITLE
Clearification about winit and gtk

### ISSUE THREAD

**Issue by ShalokShalom** [tauri-apps/tao#509]
The text at the end of the readme suggests me, that gtk is planned to be outfaded. 

Is this true, and is there a timeframe for this?

How high on the priority list is this?

thanks

**Comment by FabianLars** [comment]
> that gtk is planned to be outfaded.

Not anytime soon, running webkit**gtk** outside of gtk is not fun or not even possible, i don't know.
The idea is to (somehow) decouple gtk from tao/winit so that we can hopefully switch wry/tauri to upstream winit again. 
A relevant issue which tracks smaller differences: https://github.com/tauri-apps/tao/issues/470.
Apart from gtk and the things mentioned in the other issue, we also need to make the other features we added like menus, trays, etc. modular (uncoupled from tao) so that we can use them with winit. This also needs changes on the winit side so it probably will take a while, but the current plan is to have that done for tauri v2.

**Comment by ShalokShalom** [comment]
But that would still leave webkitgtk?

Even with the OpenGL renderer?

The @Servo project tried to develop an interface just in web languages and with WebGL rendering. 

I really want a Qt based solution.
Or really anything else than GTK. 

There is a Qt port of WebKit as well.

**Comment by FabianLars** [comment]
both gtk and webkitgtk will stay with us, yes. It just won't be part of winit directly.

i don't know about the far future, but for now opengl windows will also use gtk windows since we swapped the whole linux platform to use that.

> I really want a Qt based solution.

Yeah no, that most likely won't happen. Using qt from rust is just the worst, i can't even describe it tbh.

> There is a Qt port of WebKit as well.

It's chromium, but whatever. Here's also another issue about that: https://github.com/tauri-apps/tauri/issues/4554

So tldr: qt won't happen. Even native x11/wayland is _currently_ more likely.

And the last thing, since you mentioned servo, there's ongoing research to turn servo into a webview and make tauri+servo happen :)

**Comment by amrbashir** [comment]
The plan is to extract functionalities from tao into their own separate crates (menus and system trays. global shortcut and clipboard will be removed), then we will make a `winit_gtk` crate which is just winit but with gtk as the linux backedn (so basically tao without the extras). I hope this clarifies the situation.
